core/src/main/scala/org/broadinstitute/dsde/rawls/webservice/AdminApiService.scala

```diff
@@ -87,6 +87,22 @@ trait AdminApiService extends HttpService with PerRequestCreator with UserInfoDi
           WorkspaceService.AdminAbortSubmission(WorkspaceName(workspaceNamespace, workspaceName), submissionId))
       }
     } ~
+    path("admin" / "realms") {
+      post { //create a realm
```

Comment:
POST for creates. I think most of the inconsistent endpoints are back from before we tried to follow the more typical REST conventions